ui/src/data_explorer/components/QueryBuilder.js

```diff
@@ -96,7 +97,33 @@ const QueryBuilder = React.createClass({
   },
 
   renderLists() {
-    const {query} = this.props
+    const {query, layout} = this.props
+
+    if (layout === 'panel') {
```

Comment:
i would suggest adding a comment here that explains that other layout styles are coming, and possibly to remove the comment once they're added; otherwise it's a bit confusing in this commit along, if this were merged, as to why there's only a single layout type supported.